library/Icingadb/Model/State.php

```diff
@@ -100,7 +101,7 @@ public function getColumns()
             'last_state_change',
             'next_check',
             'next_update',
-            'affects_children'
+            'affects_children' => new Expression("'y'")
```

Comment:
if you update your state using the current dependency writer (with an updated db schema of course) you wouldn't require this